setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+with open("Readme.md", "r") as readme_file:
+    readme = readme_file.read()
+
+setup(
+    name="converter",
+    version="1.0.1",
+    
+    author="Maria Maksimova",
+    author_email="maryia.maksimava@yahoo.com",
+    
+    description="A package to converting temperature from Celsius to Fahrenheit and vise versa",
```

Comment:
```suggestion
    description="A package for converting temperature from Celsius to Fahrenheit and vise versa",
```

Reply:
Added in [setup.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)